src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4954,13 +4961,25 @@ public override bool PreVisit(CallNode node)
                         _txb.AddVolatileVariables(args[i], volatileVariables);
                     }
 
+                    var isIdentifier =
+                        args[i] is FirstNameNode &&
+                        _features.HasFlag(Features.SupportIdentifiers) &&
+                        maybeFunc.IsIdentifierParam(i);
+
                     // Use the new scope only for lambda args.
-                    _currentScope = (maybeFunc.IsLambdaParam(i) && scopeInfo.AppliesToArgument(i)) ? scopeNew : scopeNew.Parent;
-                    args[i].Accept(this);
+                    _currentScope = ((maybeFunc.IsLambdaParam(i) || isIdentifier) && scopeInfo.AppliesToArgument(i)) ? scopeNew : scopeNew.Parent;
 
-                    _txb.AddVolatileVariables(node, _txb.GetVolatileVariables(args[i]));
+                    if (!isIdentifier)
+                    {
+                        args[i].Accept(this);
+                        _txb.AddVolatileVariables(node, _txb.GetVolatileVariables(args[i]));
+                        argTypes[i] = _txb.GetType(args[i]);
+                    }
+                    else
+                    {
+                        argTypes[i] = DType.String;
```

Comment:
> DType.String; [](http://example.com/codeflow?start=37&length=14)

why string? 
Does anybody use this value?

Reply:
Yes!
We need a valid type and you didn't want to add DType.Identifier

Reply:
Consider: DType.Unknown, since we don't actually care what it is. DType.String may lead to someone depending on it.